tests/testthat/test-read.R

```diff
@@ -1,5 +1,6 @@
 # This tests the readH5AD function (and by implication, SCE2AnnData).
 library(SummarizedExperiment)
+library(anndata) # import to see whether anndata breaks zellkonverter or not
```

Comment:
I would probably move this to a specific incompatibility test (maybe using `withr::with_package()` or some base equivalent)